Documentation/RAC5MigrationGuide.md

```diff
@@ -0,0 +1,63 @@
+# RAC 5.0 Migration Guide
+
+* [Repository Split](#repository-split)
+	* [ReactiveCocoa](#reactivecocoa)
+	* [ReactiveSwift](#reactiveswift)
+	* [ReactiveObjC](#reactiveobjc)
+	* [ReactiveObjCBridge](#reactiveobjcbridge)
+* [API Names](#api-names)
+* [Signal](#signal)
+	* [Lifetime Semantics](#lifetime-semantics)
+* [SignalProducer](#signalproducer)
+* [Properties](#properties)
+* [Atomic](#atomic)
+
+## Repository Split
+As part of RAC 5.0, ReactiveCocoa has been split into multiple repositories. _The rationale for this change is explained below._
```

Comment:
I also think it would be helpful to include a link to jump down to the rationale.